kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -145,30 +144,25 @@
         // creates array of words in filter, removes empty strings
         const searchFilter = this.searchFilter.split(' ').filter(Boolean).map(
           // returns an array of search parameters, ignoring case
-          (query) => new RegExp(query, 'i'));
+          (query) => {
+            const escapedQuery = query.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
```

Comment:
The only reason that's there is to escape `?` and other meaningful characters from the input. 

The regex is used later in line 175 to test for matches in `fullname` and `username`. @christianmemije pointed out that using regex's `.test` when the input contains a character like `?` would break search, so this was a bugfix.

re: "I can't think of a good reason to be using regex here" - When I came up with this a few months back, I thought I had a few use cases where substring matching wouldn't work. I can't remember what they were, and I'm no longer convinced they exist, so I'll switch over to substring :sweat_smile:.